### ISSUE TITLE
Material-UI CSS not injected

### ISSUE THREAD

**Issue by saraivinha85** [Silind-Software/direflow#106]
**Describe the bug**  
Material-UI CSS not injected when components are rendered after first mount.

**To reproduce**  
Working code:

```
render() {
        const buttons = ['Hello'].map((p) => <Button variant="contained" color="primary" key={p}>p</Button>)

        return (
            <div style={styles.root} > 
                {buttons}
            </div>
        )
    }
````

Not working code (labels are injected later using redux):

```   
render() {
            const { labels = [] } = this.props.labels
        const buttons = labels.map((p) => <Button variant="contained" color="primary" key={p}>p</Button>)

        return (
            <div style={styles.root} > 
                    {buttons}
            </div>
        )
    }
```

**Expected behavior**  
The buttons should appear styled.

**Comment by Silind** [comment]
Thanks for creating this issue.
We will look into this as soon as possible.

**Comment by saraivinha85** [comment]
Take a look at this example:
https://github.com/saraivinha85/direflow-MUI-CSS-bug/

**Comment by Silind** [comment]
Hi @saraivinha85 
This issue has been fixed in version 3.4.5.
I have verified it on your reproduction repo.

Thanks for creating this issue :+1:
